airbyte-integrations/connectors/source-salesforce/unit_tests/api_test.py

```diff
@@ -33,6 +33,26 @@ def test_bulk_sync_creation_failed(stream_config, stream_api):
         assert err.value.response.json()[0]["message"] == "test_error"
 
 
+def test_bulk_stream_fallback_to_rest(mocker, requests_mock, stream_config, stream_api):
```

Comment:
I don't get how this test is testing that a fallback to REST happens.

Reply:
In case it does not fallback to REST, an exception would be raised. I'll add a comment explaining that